package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "flowcrypt-ios",
+  "version": "1.0.0",
+  "description": "Flowcrypt iOS app",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepare": "husky install"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FlowCrypt/flowcrypt-ios.git"
+  },
+  "author": "Flowcrypt",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FlowCrypt/flowcrypt-ios/issues"
+  },
+  "homepage": "https://github.com/FlowCrypt/flowcrypt-ios#readme",
+  "devDependencies": {
+    "husky": "^8.0.1",
+    "lint-staged": "^12.4.3"
+  },
+  "lint-staged": {
+    "Core/*.{js,ts}": [
+      "cd Core && npm test",
+      "cd Core && npm run tslint",
+      "cd Core && npm run eslint"
```

Comment:
It's not good idea to run `npm test` for every commits.
Here are options which I can suggest.

1. Do not run commit hook for `Core` and keep `Typescript tests` in semaphoreci.
2. Run `npm run tslint` and `npm run eslint` for staged files for every commits & run `npm test` for every pushes. And delete `Typescript tests` in semaphoreci. (I prefer this one)
3. Run `npm test`, `npm run tslint`, `npm run eslint` for every commits. (This seems bad idea)

Open to discussion @tomholub, @sosnovsky

Reply:
> Run npm run tslint and npm run eslint for staged files for every commits & run npm test for every pushes. And delete Typescript tests in semaphoreci. (I prefer this one)

I guess this but why delete tests in semaphoreci? Let them run there too, it won't hurt.

Reply:
It won't hurt but it won't be necessary neither.

Reply:
All changes in git will be checked before every commits and Typescript tests on semaphoreci won't be necessary